test/e2e/basic_test.go

```diff
@@ -108,21 +108,21 @@ func TestNatsUpgrade(t *testing.T) {
 		}
 	}()
 
-	err = e2eutil.WaitSizeAndVersionReached(t, f.KubeClient, "v1.0.0", 3, 6, testNats)
+	err = e2eutil.WaitSizeAndVersionReached(t, f.KubeClient, "1.0.0", 3, 6, testNats)
 	if err != nil {
 		t.Fatalf("failed to create 3 members NATS cluster: %v", err)
 	}
 
 	updateFunc := func(cl *spec.NatsCluster) {
-		cl = e2eutil.ClusterWithVersion(cl, "v1.0.2")
+		cl = e2eutil.ClusterWithVersion(cl, "1.0.2")
 	}
 	_, err = e2eutil.UpdateCluster(f.CRClient, testNats, 10, updateFunc)
 	if err != nil {
 		t.Fatalf("fail to update cluster version: %v", err)
 	}
 
 	// We have seen in k8s 1.7.1 env it took 35s for the pod to restart with the new image.
-	err = e2eutil.WaitSizeAndVersionReached(t, f.KubeClient, "v1.0.2", 3, 10, testNats)
+	err = e2eutil.WaitSizeAndVersionReached(t, f.KubeClient, "1.0.2", 3, 10, testNats)
 	if err != nil {
 		t.Fatalf("failed to wait new version NATS cluster: %v", err)
 	}
```

Comment:
It is, indeed, but in Docker Hub (https://hub.docker.com/r/library/nats/tags/) they have no prefix so I ended up removing it. 🤔